runnc-cli/delete.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/urfave/cli"
+)
+
+var deleteCommand = cli.Command{
+	Name:  "delete",
+	Usage: "delete any resources held by the container often used with detached container",
+	ArgsUsage: `<container-id>
+
+Where "<container-id>" is the name for the instance of the container.
+
+EXAMPLE:
+For example, if the container id is "ubuntu01" and runc list currently shows the
```

Comment:
should the runc mentioned here, on line 19 and line 21 actually be runnc?

Reply:
Yup! Thanks - i've fixed that in commit: abc61317ed3e9aaba87787b1ef774adfeaac6687